src/ModifiedHankelFunctionsOfOrderOneThird.jl

```diff
@@ -34,8 +34,10 @@ const dvec = @SVector [d(i) for i = 0:NUMTERMS]
 
 Cterm(m) = 9*(2m - 1)^2 - 4
 C(m) = convert(Float64, prod(Cterm, 1:m)/(2^(4m)*3^m*factorial(m)))
-const Cvec = @SVector [C(big(i)) for i = 1:NUMTERMS]  # `big` required for `factorial`
-
+# `big` required for `factorial`; also, this begins with 1 to simplify evalpoly below
+const Cvec = @SVector [i == 0 ? 1.0 : C(big(i)) for i = 0:NUMTERMS]
+# begin with 0 to simplify evalpoly; Cvec[i+1] to offset from the 0 term of Cvec
+const Cveci = @SVector [i == 0 ? 0.0 : Cvec[i+1]*i for i = 0:NUMTERMS]
```

Comment:
I don't think `evalpoly` has a specialized method for `SVector`, so you may be better off using `Tuple(@SVector[...])` here — that will allow the compiler to inline the `evalpoly` call.

Also, there's no point in making the first coefficient zero, since you can just factor that out — for example, rather than `evalpoly(x, (0, 1, 2, 3))` you can do `x * evalpoly(x, (1, 2, 3))`.

Reply:
Oh, I see, you are using `Cvec.data` already below to get the tuple.

Reply:
(Actually, I just checked, and it seems like the compiler is already smart enough to inline/unroll `evalpoly` acting on an `SVector`.)